src/EA.Weee.RequestHandlers/AatfEvidence/TransferTonnagesValidator.cs

```diff
@@ -34,7 +34,7 @@ await evidenceDataAccess.GetTonnageByIds(transferValues.Select(t => t.TransferTo
                 //TODO: this needs to be updated to only exclude rejected, voided notes
```

Comment:
Remove this comment... I also think there are unit test / integration tests that need to be re-instated. Search for "Re-instate when available tonnages i" in the code

Reply:
I believe this is done now